sentry/patch.py

```diff
@@ -0,0 +1,93 @@
+# Copyright 2022 Akretion (https://www.akretion.com).
+# @author Sébastien BEAU <sebastien.beau@akretion.com>
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+
+# Monkey patch odoo method in order to track all database
```

Comment:
What about using manifest hooks?

In order to run this code only if the module is already installed

Also, you could remove the try-except-ImportError section using this way